server/app/models/LifecycleStage.java

```diff
@@ -17,6 +18,13 @@ public enum LifecycleStage {
 
   private final String stage;
 
+  private static ImmutableMap<LifecycleStage, String> descriptions =
+      ImmutableMap.of(
+          LifecycleStage.DRAFT, "NOT_SUBMITTED",
+          LifecycleStage.ACTIVE, "CURRENT",
+          LifecycleStage.OBSOLETE, "OBSOLETE",
```

Comment:
suggestion: If we move this to services/export/enums, and then do the mapping from LifecycleStage to this in the json export code, then this will only ever be used in the context of the export. That means we can just throw `NotImplemented` if the LifecycleStage is anything other than Active or Obsolete, since the export code should never even see Draft or Deleted applications.